rhea/system/hwtypes.py

```diff
@@ -39,11 +39,13 @@ def __call__(self, val=0):
 
 def Signals(sigtype, num_sigs):
     """ Create a list of signals
-    Arguments:
-        sigtype (bool, intbv): The type to create a Signal from.
-        num_sigs (int): The number of signals to create in the list
-    Returns:
-        sigs: a list of signals all of type `sigtype`
+
+    | Arguments:
```

Comment:
I don't understand the change here?  I have been following the docstring format outlined here (suggested google format):
https://pypi.python.org/pypi/sphinxcontrib-napoleon
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#google-vs-numpy

Why add the"|" character?